tests/Functional/AdminTest.php

```diff
@@ -68,27 +69,39 @@ public function setUp()
         $this->dm = $this->getContainer()->get('doctrine_phpcr.odm.default_document_manager');
     }
 
-    public function testAdmin()
+    public function getAdmin()
     {
-        $adminGroups = $this->pool->getAdminGroups();
+        $pool = $this->getContainer()->get('sonata.admin.pool');
+        $adminGroups = $pool->getAdminGroups();
         $admins = [];
 
         foreach (array_keys($adminGroups) as $adminName) {
-            $admins = array_merge($admins, $this->pool->getAdminsByGroup($adminName));
+            $admins = array_merge($admins, $pool->getAdminsByGroup($adminName));
         }
 
-        foreach ($admins as $admin) {
-            $this->doTestReachableAdminRoutes($admin);
-        }
+        return $admins;
```

Comment:
$admins is an one dimensional array of `AdminInterface` which is not a valid dataprovider.